Admin/ORM/MediaAdmin.php

```diff
@@ -36,7 +36,7 @@ protected function configureDatagridFilters(DatagridMapper $datagridMapper)
             ->add('context', null, array(
                 'show_filter' => $this->getPersistentParameter('hide_context') !== true,
             ), 'choice', $options)
-            ->add('category', null, array(
+            ->add('category', 'doctrine_orm_choice', array(
```

Comment:
Ok, thanks, let's see than what went wrong and why it does not work...

BTW, I just went to try out demo app: http://rftsgb.demo.sonata-project.org/admin/app/media-media/list?category=19

You can reproduce bug there as well.